src/DynamoCore/Models/RecordableCommands.cs

```diff
@@ -924,14 +924,28 @@ public class UpdateModelValueCommand : RecordableCommand
 
             #region Public Class Methods
 
-            public UpdateModelValueCommand(Guid modelGuid, string name, string value)
-                : this(new[] {modelGuid}, name, value)
+            /// <summary>
+            /// </summary>
+            /// <param name="modelGuid"></param>
+            /// <param name="name"></param>
+            /// <param name="value"></param>
+            /// <param name="workspaceModel">If workspace model is null, this command will be sent to the current workspace</param>
+            public UpdateModelValueCommand(Guid modelGuid, string name, string value, WorkspaceModel workspaceModel = null)
+                : this(new[] {modelGuid}, name, value, workspaceModel)
```

Comment:
Hi @ke-yu, if I may suggest a preference, can we update the signature of this constructor (and the one after this) to be:

```csharp
public UpdateModelValueCommand(Guid workspaceGuid, Guid modelGuid, string name, string value)
{
    if (workspaceGuid == System.Guid.Empty)
        throw new ArgumentNullException("workspaceGuid");
}
```

To me this is more than a personal preference, as it also accomplishes the following:

1. Having a `workspaceGuid` come before `modelGuid` tells a reader that the model is a smaller element that is contained in a workspace.
2. Using `System.Guid` instead of `WorkspaceModel` as a more serialization friendly data type, and it aligns with most other recordable commands.

In order not to update tons of other callers of this class, we can retain the two original constructors which does not take `WorkspaceModel.Guid` (so the old tests continue tto work with current workspace). I'm not sure if we can mark these original constructors as obsolete?